### PR TITLE
History of change in section preview

### LINKED ISSUE #1062: History of change in section preview

### PR DESCRIPTION
**Pull request by Katrin-kudryash**
## PR includes

-   [x] Refactor
-   [x] Feature

## Related issues

Resolve #1062

### CODE REVIEW

**Review by tsumo — CHANGES_REQUESTED** [review]
Please delete SectionHistoryPage

File: src/pages/interviews/[interviewId]/sections/[sectionId]/history.tsx [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { SectionHistoryPage } from '../../../../../components/SectionHistoryPage/SectionHistoryPage';
```
Comment (tsumo) [inline src/pages/interviews/[interviewId]/sections/[sectionId]/history.tsx:1]: This file is also can be deleted

**Review by tsumo — CHANGES_REQUESTED** [review]
Also `pageHrefs.interviewSectionHistory` is not needed anymore

**Comment by tsumo** [comment]
> Also `pageHrefs.interviewSectionHistory` is not needed anymore

@Katrin-kudryash

**Review by asabotovich — APPROVED** [review]

**Pull request merged**